articles/api-management/api-management-using-with-vnet.md

```diff
@@ -132,6 +132,8 @@ When an API Management service instance is hosted in a VNET, the ports in the fo
 
 + **Metrics and Health Monitoring**: Outbound network connectivity to Azure Monitoring endpoints, which resolve under the following domains:
 
++ **Regional Service Tags**": NSG rules allowing outbound connectivity to Storage, SQL, and EventHubs service tags may use the regional versions of those tags corresponding to the region containing the API Management instance (for example, Storage.WestUS for an API Management instance in the West US region). In multi-region deployments, the NSG in each region should allow traffic to the service tags for that region.
```

Comment:
This is not accurate. The analytics aggregation runs only in the master region. So the additional region, need to communicate with the Storage Queue in the master region. I say we remove this bulletin on Regional Service Tag, and add when fully tested.

Reply:
@solankisamir thanks for checking, so this info from Kedar is wrong?
"Traffic to SQL, Storage and EH would be regional for APIM except when APIM's multi-region feature is used.  In that case the traffic would go to SQL from each of the region that APIM was added."
https://www.yammer.com/microsoft.com/#/Threads/show?threadId=67220167319552

I could certainly change it to indicate that extra regions need to communicate with Storage in the main region if that's useful.
Locking the NSGs down further than what is listed in the doc is a pretty common issue with our Japanese customers so I would like to get this doc updated to show the most restrictive settings possible.